### PR TITLE
Add assigning, removing charges to and from products and services

### PR DESCRIPTION
**Pull request by AluminumAlman**
additionally fixes bug in order charge which allowed super admins not assigned to a merchant assign order charges to products and services between merchants

### CODE REVIEW

File: 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public ChargeController(ChargeService chargeService) {
         this.chargeService = chargeService;
     }
 
-    @GetMapping("/merchant")
+    @GetMapping("/charges/merchant")
     @Operation(summary = "Get all charges by merchant ID")
```
Comment (Dalt0nik) [inline 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java:26]: remove merchant from path

File: 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java [hunk 2]
```diff
@@ -9,11 +9,11 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
 import java.util.UUID;
 
 @Slf4j
 @RestController
-@RequestMapping("/charges")
 @OneOf(fields = {"percent", "amount"})
```
Comment (HubertasVin) [inline 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java:17]: Kodėl ištrynei @RequestMapping?

File: 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeService.java [hunk 3]
```diff
@@ -138,6 +235,13 @@ public List<Charge> getAllEntitiesById(List<UUID> chargeIds) {
 
     // *** Helper methods ***
 
+    public Charge getChargeEntityById(UUID chargeId) {
+        Charge charge =  chargeRepository.findById(chargeId).orElseThrow(() -> new ResourceNotFoundException(
+                ResourceType.CHARGE,
+                chargeId.toString()));
+        return charge;
+    }
+
```
Comment (HubertasVin) [inline 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeService.java:243]: Galima nekurti naujo kintamojo ir tiesiog iš karto returninti findById rezultatą

**Review by HubertasVin — CHANGES_REQUESTED** [review]

File: 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java (hunk 2 above)
Comment (AluminumAlman) [reply 2lab/POS/src/main/java/com/team1206/pos/payments/charge/ChargeController.java:17]: Nes kiti endpointai turi path-ą prasidedančiu su product ir service

**Review by HubertasVin — APPROVED** [review]

**Pull request merged**